### PR TITLE
Fix NOTIFY_INBOX_ONLY value in workspace deployment

### PR DESCRIPTION
**Pull request by 0xtejas**
Fixes issue #61

### CODE REVIEW

File: kube/workspace/workspace-deployment.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
             - name: TRANSACTOR_URL
               value: ws://transactor:3333;ws://localhost:3333
             - name: NOTIFY_INBOX_ONLY
```
Comment (aonnikov) [inline kube/workspace/workspace-deployment.yaml:50]: Could you please remove it at all? This env variable is not used anymore

**Review by aonnikov — CHANGES_REQUESTED** [review]

File: kube/workspace/workspace-deployment.yaml (hunk 1 above)
Comment (0xtejas) [reply kube/workspace/workspace-deployment.yaml]: Done

File: kube/workspace/workspace-deployment.yaml [hunk 2]
```diff
@@ -45,10 +45,7 @@ spec:
                 secretKeyRef:
                   name: huly-secret
                   key: SERVER_SECRET
-            - name: TRANSACTOR_URL
-              value: ws://transactor:3333;ws://localhost:3333
-            - name: NOTIFY_INBOX_ONLY
-              value: true
+              value: "true"
```
Comment (aonnikov) [inline kube/workspace/workspace-deployment.yaml:48]: value not needed?

Comment (0xtejas) [reply kube/workspace/workspace-deployment.yaml]: Did not notice that I left that 😅 , I've updated it.

**Review by aonnikov — APPROVED** [review]

**Pull request merged**